### ISSUE TITLE
FreeBSD port: almost there

### ISSUE THREAD

**Issue by nunotexbsd** [johnmehr/gitup#14]
Hello,

Thanks for your work on svnup.

I don't like git,
I use gitup!

[252220](https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=252220)

Thanks,

Nuno Teixeira

**Comment by nunotexbsd** [comment]
0.87 tag vanished?
How can it be?

**Comment by johnmehr** [comment]
The bugfix for issue #12 I committed on the 25th didn't work in all cases so I deleted the tag and committed a better fix but I forgot to restore the tag.  Sorry about that!  I just added the tag back.

Thank you for setting up the port!  I have a couple of questions about using the name "gitup" -- I did a quick search and it's there are a couple of projects that already use that name (and there is also a camera brand called "GitUp") and I'm wondering if we should come up with a different name to avoid any possible issues before the port is created.  Thoughts?

Thank you!

**Comment by nunotexbsd** [comment]
Hi!

IMO "gitup" is already registered at your github project and there is no problem to FreeBSD ports because there is no similar port names there.
"gitup" and "svnup" are good quality names because they short and intuitive.

Could you participate on my PR [252220](https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=252220) ? I have a big picture question:

"dependency-free" means what?
Should it be -lprivateucl (base) instead of textproc/libucl?"
(related to lib depends on textproc/libucl, but it works fine with base -lprivateucl (libprivateucl.so.1 => /usr/lib/libprivateucl.so.1)"

Thanks,

Nuno

**Comment by johnmehr** [comment]
I tried to reset my password at bugs.freebsd.org and I haven't received the email yet. :(

"Dependency-free" means that it's a stand alone application and doesn't rely on anything not included in the base system.

It should definitely use -libprivateucl (I don't have textproc/libucl installed and I did all development work using the base library).

It looks likely that I'll have to rename it to avoid confusion with existing projects named "gitup".
